reference/library/src/morpheus-master/bootstrap-sass-3.3.6/assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -80,7 +80,7 @@ $headings-color:          inherit !default;
 
 // [converter] If $bootstrap-sass-asset-helper if used, provide path relative to the assets load path.
 // [converter] This is because some asset helpers, such as Sprockets, do not work with file-relative paths.
-$icon-font-path: if($bootstrap-sass-asset-helper, "bootstrap/", "../fonts/bootstrap/") !default;
+$icon-font-path: if($bootstrap-sass-asset-helper, "bootstrap/", "../morpheus-default/fonts/bootstrap/") !default;
```

Comment:
Hi, Sam ... That "morpheus-default" is not mandatory, you can change it as a property or if your default skin is named from another way. 

There's no a magic trick to put that property to sass so I've been thinking if you may add that literal string to a variable on _defaults.scss so if everyone wants to change the skin it would load from it's own skin, maybe `skinName` or something like that.

I'll update documentation, of course.